control-plane/agents/src/bin/core/controller/scheduling/volume_policy/mod.rs

```diff
@@ -59,4 +59,7 @@ impl DefaultBasePolicy {
             .filter(pool::PoolBaseFilters::capacity)
             .filter(pool::PoolBaseFilters::min_free_space)
     }
+    fn filter_resize(request: ResizeVolumeReplicas) -> ResizeVolumeReplicas {
+        request.filter(pool::PoolBaseFilters::min_free_space_repl_resize)
```

Comment:
missing volume overcommit check in min_free_space_repl_resize, probably we could instead modify existing min_free_space check which also checks for snapshots
also missing pool overcommit checks

Reply:
You mean modify the  `SimplePolicy:: min_free_space_repl_resize` ? 
Need to write `ReplicaResizePoolsContext` versions of `SimplePolicy:: min_free_space_repl_resize` and `SimplePolicy::overcommit` perhaps, which will be applied after `PoolBaseFilters::min_free_space_repl_resize` ? Existing ones expect `GetSuitablePoolsContext` request.

Reply:
Alright, then we need to add a new one, can add to min_free_space_repl_resize then; might need some refactor to be able to reuse code

Reply:
PTAL now